dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx

```diff
@@ -163,10 +163,14 @@ export const ThreeTierChoiceCards = ({
 					({ product, label, benefitsLabel, benefits, pill }) => {
 						const { supportTier } = product;
 
-						const selected =
-							selectedProduct.supportTier === supportTier;
+						const selected = selectedProduct === product;
```

Comment:
this means we compare both the supportTier and ratePlan

Reply:
though I've realised this doesn't always work as it's comparing object references! Fix incoming

Reply:
fix: https://github.com/guardian/dotcom-rendering/pull/13984/commits/1946406bf2099065fc51eb522573f1978bd2949e